.mergify.yml

```
@@ -1,8 +1,11 @@
 pull_request_rules:
   - name: instruct bors to merge dependabot PRs with passing tests
     conditions:
-      - "status-success=ci/circleci: build_test"
-      - "status-success=ci/circleci: static_analysis"
+      - "status-success=CI / static_analysis (push)"
```

📣 
```suggestion
      - "status-success=static_analysis"
```
This is how it should be named.
See comit-sdk config as proof/example.